lineman/app/components/group_page/discussions_card/discussions_card.coffee

```diff
@@ -13,7 +14,13 @@ angular.module('loomioApp').directive 'discussionsCard', ->
         from:     $scope.loaded
         per:      $scope.perPage
       $scope.loaded += $scope.perPage
-      Records.discussions.fetchByGroup options
+      Records.discussions.fetchByGroup(options).then (data) ->
+        if data.discussions?
```

Comment:
This logic confuses me a little, I think because there's a lot of negatives.

What if it was allDiscussionsLoaded, which defaults to false, and is set to true if `(data.discussions or []) < $scope.perPage`?